app/helpers/application_helper.rb

```diff
@@ -8,4 +11,32 @@ def full_title(page_title = "")
       page_title + " | " + base_title
     end
   end
+
+  def email_slack_invite(user_email)
+    # Get the channel ID from https://slack.com/api/channels.list?token=<YOUR_TOKEN>
+    channels = ""
+
+    # Get the auth token from: https://api.slack.com/web
+    token = ""
+
+    # Construct the API request URL
+    # This is not documented by Slack, and may not work forever
+    # As it is an unofficial API endpoint, use at your own risk!
+    url = "https://slack.com/api/users.admin.invite"
+    url += "?channels=#{channels}"
+    url += "&set_active=true"
+    url += "&_attempts=1"
+    url += "&token=#{token}"
+    url += "&email=#{URI.escape(user_email)}"
```

Comment:
Using [`HTTP::URI.build`](http://ruby-doc.org/stdlib-2.0.0/libdoc/uri/rdoc/URI/HTTP.html#method-c-build) instead of concatenating a string could lead to more readable and maintainable code.